scripts/release.sh

```diff
@@ -59,5 +64,11 @@ for runtime in "${runtimes[@]}"; do
     artifacts+=("${artifact_name}")
 done
 
-# Create a pre-release using GitHub CLI
-gh release create --title "${tag_name}" --draft "${tag_name}" "${artifacts[@]}" "${checksum_name}"
+# Create a release using GitHub CLI
+if [[ "$tag_name" == "-" ]]; then
+    # v1.0.0-rc.1 is a pre-release
+    gh release create --title "${tag_name}" --prerelease "${tag_name}" "${artifacts[@]}" "${checksum_name}"
```

Comment:
We still need to make it a draft.
```suggestion
    gh release create --title "${tag_name}" --draft --prerelease "${tag_name}" "${artifacts[@]}" "${checksum_name}"
```

Reply:
Updated.